docs/core/ADMIN_SECURITY.md

```
@@ -0,0 +1,244 @@
+
+### Step 1: Set Admin Credentials in .env
+
+Add to your `.env` file or Coolify environment variables:
+
+```bash
+SMS_BRIDGE_ADMIN_USERNAME=admin
+SMS_BRIDGE_ADMIN_PASSWORD=YourVeryStrongPassword123!
+```
+
+**Generate a strong password:**
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)